src/aat/resources/features/F-140 - Get Linked Case V2 External/S-140.3.td.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_guid_": "S-140.3",
+  "_extends_": "Get_Linked_Cases__Base",
+  "title": "should get 404 when case reference exist but not authorised",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains a valid user authorisation token without access to the operation",
+    "contains a case reference that does not exist",
+    "contains an HTTP-404 Not Found"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][childContexts][Unauthorised_Case_Creation_Data][testData][actualResponse][body][id]}"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "body": {
+      "exception": "uk.gov.hmcts.ccd.domain.service.getcase.CaseNotFoundException",
+      "message": "No case found for reference: ${[scenarioContext][childContexts][Unauthorised_Case_Creation_Data][testData][actualResponse][body][id]}",
+      "path": "/getLinkedCases/${[scenarioContext][childContexts][Unauthorised_Case_Creation_Data][testData][actualResponse][body][id]}",
+      "details": null,
```

Comment:
NB: should switch scenario to use `[Standard_Full_Case_Creation_Data]` as per the other tests but override `users.invokingUser` in this call to one from another jurisdiction (i.e. one without access to case)

Reply:
Fixed in [54f1d4f](https://github.com/hmcts/ccd-data-store-api/pull/1920/commits/54f1d4fcf3faac512d1a6278a71e82b99881cce4)